ui/analyse/src/study/relay/relayCtrl.ts

```diff
@@ -42,7 +41,7 @@ export default class RelayCtrl {
       ? new RelayTeams(id, this.multiCloudEval, setChapter, this.roundPath, redraw)
       : undefined;
     this.leaderboard = data.tour.leaderboard ? new RelayLeaderboard(data.tour.id, redraw) : undefined;
-    setInterval(this.redraw, 1000);
+    setInterval(() => this.redraw(true), 1000);
```

Comment:
The interval is here to guarantee that the clocks on screen are ticking. But there are a lot more events that trigger redraws, such as incoming websocket messages for new moves, clocks, chat messages, etc.